docs/HowToInstall.md

```diff
@@ -0,0 +1,99 @@
+# How to install?
+
+## Prerequisite
+
+ONE-vscode is an [extension](https://code.visualstudio.com/docs/editor/extension-marketplace) of [vscode](https://code.visualstudio.com/). To use ONE-vscode, you should install vscode first.
+
+### vscode and its extensions
+
+Let's install vscode.
+- You can download vscode [HERE](https://code.visualstudio.com/download)
+- Official document: [setup](https://code.visualstudio.com/docs/setup/setup-overview)
+- (You should also install npm for vscode)
+
+### ssh extension in vscode
+
+Now [ONE](https://github.com/Samsung/ONE) officially releases on linux/ubuntu only. If your development environment is on Windows, you should use ssh extension in vscode. Let's follow [tutorial](https://code.visualstudio.com/docs/remote/ssh-tutorial).
+
+### environment setting on linux/ubuntu
+
+Linux machine must have `aptitude` and `docker` packages.
```

Comment:
```suggestion
Linux machine must have `aptitude`.
```

and `aptitude` is installed by toolchain view. 
@jyoungyun, could you check if we need this statement?

Reply:
After @jyoungyun 's check, I'll merge.